### PR TITLE
Display pictures from PictureField in some email notifications

### PR DESCRIPTION
**Pull request by SegiNyn**
The picture is displayed in the registration creation and modification email notifications since those are the places that show field data.

### CODE REVIEW

File: indico/vendor/django_mail/message.py [hunk 1]
```diff
@@ -428,6 +428,7 @@ def _create_attachment(self, filename, content, mimetype=None):
                 filename.encode('ascii')
             except UnicodeEncodeError:
                 filename = ('utf-8', '', filename)
+            attachment.add_header('Content-ID', filename)
```
Comment (ThiefMaster) [inline indico/vendor/django_mail/message.py:431]: These files cannot be modified, they are 1:1 copies from another project.

Comment (ThiefMaster) [reply indico/vendor/django_mail/message.py:431]: Maybe something on https://stackoverflow.com/questions/3787755/django-how-to-send-html-emails-with-embedded-images helps

Comment (SegiNyn) [reply indico/vendor/django_mail/message.py:431]: perfect, I thought I could escape using MIMEImage

Comment (ThiefMaster) [reply indico/vendor/django_mail/message.py:431]: BTW, filenames are not guaranteed to be unique so they do not make good cids

Comment (SegiNyn) [reply indico/vendor/django_mail/message.py]: Yes, they're not unique so I was passing the html_field_name + filename as the new filename which I think should be unique

Comment (SegiNyn) [reply indico/vendor/django_mail/message.py]: Hi @ThiefMaster, one thing missing from this pr is the display of old image and new image side by side in the case of registration modification as you do for the other fields. Is it necessary to display the previous for images as well?

Comment (ThiefMaster) [reply indico/vendor/django_mail/message.py]: i don't think we need that...

Comment (SegiNyn) [reply indico/vendor/django_mail/message.py]: okay then.

**Comment by SegiNyn** [comment]
Hi @ThiefMaster, it looks good, I just noticed that the yellow highlight on new texts were not displaying so I fixed that

**Review by ThiefMaster — APPROVED** [review]

**Pull request merged**